### PR TITLE
UsereXperience improvements

### PR DESCRIPTION
**Pull request by Maciej-GBS**
Hello, nice work Matt :+1: 

I have included couple of changes that are here to make it easier and faster to work with this local installation:

- newbie first configuration - secrets are now set to `minio` and `miniokey` by default (for some reason dr-coach fails to connect with S3 if different keys are set); local mode is enabled, assuming that most new linux users do not use tmux
- extended gitignore
- docker-compose - changed default image to cpu-avx2 (as this robomaker version will be most often used on modern machines); fixed a typo in volumes? (double `//`)
- shortcuts - created symbolic links to hyperparams.json and minio/custom_files
- rl_deepracer_coach_robomaker.py - now loads all hyperparameters from json file hyperparams.json located in the same directory, which is nicer and easier to edit; also allows to enable pretrained mode with just one flag
- README now contains information about connecting via vncviewer
- start-training.sh - surpressed warning from deprecated `-x` option, updated to new syntax
- local-copy.sh - backup script for exporting model data from s3 bucket to selected directory (by default `../models/`)
- mk-model.sh - creates output.tar.gz file inside any folder that contains model data (necessary for uploading into physical car); works best with local-copy.sh because working on bucket/current requires sudo privileges
- utilities/delete-last.c - linux program designed to bypass sudo prompt every time you run delete_last_run.sh, all instructions how to compile and set up are inside the file

This configuration was tested on my ubuntu18.04 and works perfectly, but I have not tested (nor modified) tmux mode.

There are some dirty changes like selected track in config files or model_metadata.json but these are easy to fix if conflicted.

### CODE REVIEW

**Comment by mattcamp** [comment]
Lots of this stuff looks really good... why did you close it?

**Comment by Maciej-GBS** [comment]
I started working on further improvements so I will reopen it soon

File: data/minio/bucket/custom_files/training_params.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-WORLD_NAME: "LGSWide"
+WORLD_NAME: "Bowtie_track"
```
Comment (mattcamp) [inline data/minio/bucket/custom_files/training_params.yaml]: Please change this to default to the Reinvent 2018 track

File: mk-model.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (mattcamp) [inline mk-model.sh:1]: The model file will actually be created automatically for you if the containers are shut down in the correct way, and the correct order, however I don't think the stop script currently does this correctly so let's leave this script here for now but will remove in near future.

File: src/rl_coach_2020_v2/hyperparams.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	"epsilon_steps": 10000,
+	"exploration_type": "categorical",
+	"loss_type": "mean squared error",
+	"lr": 0.0003,
+	"num_episodes_between_training": 20,
+	"num_epochs": 10,
+	"stack_size": 1,
+	"term_cond_avg_score": 100000.0,
+	"term_cond_max_episodes": 10000,
+	"pretrained": 0
```
Comment (mattcamp) [inline src/rl_coach_2020_v2/hyperparams.json]: Can this be true/false please to be more user friendly?

File: README.md [hunk 4]
```diff
@@ -46,7 +46,7 @@ The video stream is available either via a web stream of via Kinesis.
```
Comment (mattcamp) [inline README.md:51]: Please also add some notes regarding the new location of where to edit the hyperparameters and the additional scripts you have added.

File: data/minio/bucket/custom_files/model_metadata.json [hunk 5]
```diff
@@ -2,43 +2,43 @@
     "action_space": [
         {
             "steering_angle": -30,
-            "speed": 1.0,
+            "speed": 1.5,
```
Comment (mattcamp) [inline data/minio/bucket/custom_files/model_metadata.json]: 1.5 is very fast for a 30 degree corner, we should provide a default action space that is definitely slow enough to actually learn to complete a lap in reasonable time.

**Review by mattcamp — CHANGES_REQUESTED** [review]

File: data/minio/bucket/custom_files/training_params.yaml (hunk 1 above)
Comment (Maciej-GBS) [reply data/minio/bucket/custom_files/training_params.yaml]: I have accepted incoming master change. Track is now LGSWide and OBJECT_AVOIDANCE mode.

File: mk-model.sh (hunk 2 above)
Comment (Maciej-GBS) [reply mk-model.sh:1]: Sounds good.

File: src/rl_coach_2020_v2/hyperparams.json (hunk 3 above)
Comment (Maciej-GBS) [reply src/rl_coach_2020_v2/hyperparams.json]: Done, it is case insensitive.

File: data/minio/bucket/custom_files/model_metadata.json (hunk 5 above)
Comment (Maciej-GBS) [reply data/minio/bucket/custom_files/model_metadata.json]: New speeds are set.

File: README.md (hunk 4 above)
Comment (Maciej-GBS) [reply README.md:51]: Should be all right now.

File: data/minio/bucket/custom_files/training_params.yaml [hunk 6]
```diff
@@ -20,6 +20,5 @@ METRIC_NAME: "TrainingRewardScore"
 CAR_COLOR: "Purple"
 TARGET_REWARD_SCORE: "None"
 NUMBER_OF_OBSTACLES: "3"
-CHANGE_START_POSITION: "true"
```
Comment (mattcamp) [inline data/minio/bucket/custom_files/training_params.yaml:23]: Why was this param removed?

Comment (Maciej-GBS) [reply data/minio/bucket/custom_files/training_params.yaml:23]: > Why was this param removed?

It was duplicated inside the file.
I should have marked this change.

Comment (Maciej-GBS) [reply data/minio/bucket/custom_files/training_params.yaml:23]: The param is still there on line 5

**Review by mattcamp — APPROVED** [review]

**Pull request merged**